index.html

```diff
@@ -191,7 +171,14 @@ <h3>MW Games</h3>
 	</ul>
   
   </p>
+</div> 
+<div id="ContactUs" class = "tabcontent">
+	<h3>Contact Us</h3>
+	<p> Contact us at insert contact info here
+	</p>
 </div>
+	<img class = logo, src = "file:///C:/Users/The%20Critic/Downloads/Logo-Final-outlines.svg", width = 150px, height = 150vw, alt = "Happy Robot">
```

Comment:
do not use absolute paths or paths that are only valid on your machine;
this needs to work when it is deployed in the cloud.
Use paths that are relative to the HTML file or absolute path from the root of the deployment.

Reply:
Sorry, but I'm not sure how to locate such an absolute path to do so.

Reply:
> Sorry, but I'm not sure how to locate such an absolute path to do so.

As I wrote above:

`main.css` has absolute path `/main.css` and relative to the `index.html` its path is `./main.css` or `main.css`.
When developing locally, do you use some kind of a web server to serve the pages you are working with?
If not, you need to start doing this, or paths above won't work on your machine...

Reply:
When developing locally, I haven't been using <link> to access the css from somewhere else. Maybe we can discuss this at some point, to make sure I don't mess this all up?

Reply:
To not mess this all up, you need to use a local web server while working on it. So, again: are you?

Reply:
I don't fully understand what this local testing server does or is supposed to do. Can we discuss this over Shabbos?

Reply:
So no, I don't think I'm currently using this.

Reply:
> I don't fully understand what this local testing server does or is supposed to do. Can we discuss this over Shabbos?

If you are allowed to discuss such topics on Shabbos - sure :)

Reply:
remove your-local-machine-specific paths